src/worker_index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
```

Comment:
i'd move this html somewhere else, perhaps both the html and `worker_main.js` can go into a `worker` folder